MinimOsd_Extra/protocols/NMEA.h

```diff
@@ -316,14 +300,15 @@ store to msg.nmea.lat
 
 
 void read_NMEA(){
+    char string[NMEA_BUF_LENGTH];
     while (Serial.available_S()) {
```

Comment:
No, I needn't this mode so it developed by request, so I don't test it. I use MAVlink only.